docs/ballerina-by-example/examples/decode-nested-bodyparts-in-multipart/decode-nested-bodyparts-in-multipart.bal

```diff
@@ -0,0 +1,79 @@
+import ballerina.net.http;
+import ballerina.mime;
+import ballerina.io;
+
+@http:configuration {basePath:"/nestedparts", port:9093}
+service<http> echo {
```

Comment:
Shall we make service name meaningful? We can go ahead with "nestedparts" as service name. so that you can remove basePath field in configuration